announcer.rb

```diff
@@ -65,15 +69,19 @@ def departures_crossed_interval(new_departures, old_departures)
     departures
   end
 
-  def soonest_departures(departures)
-    departure_attrs = []
-    departures.each_pair do |stop_id, route_directions|
-      route_directions.each_pair do |(route_id, sign), trips|
-        departure_attrs << { route_id: route_id, headsign: sign,
-                             stop_id: stop_id, interval: trips.values.min }
-      end
+  def issue_body
+    "Reported on #{Time.now.strftime '%A, %B %-e, %Y at %-l:%m %P'}."
```

Comment:
Too often I forget that the things you write in the `strftime` argument don't *have* to be `%`-strings.